skate-plugin/src/main/resources/messages/skateBundle.properties

```
@@ -0,0 +1,5 @@
+skate.configuration.title=Skate Settings
+skate.configuration.choosePath.title=Choose file:
```

nit: titles don't have trailing colons

```suggestion
skate.configuration.choosePath.title=Choose file
```